### ISSUE TITLE
[HELP] how to find external keyboard

### ISSUE THREAD

**Issue by vinay-keshava** [maricn/interception-vimproved#10]
Hello
i was able to build and test out interception-vimproved and works good for the built-in keyboard of the laptop
but the keybindings doesn't work same on a external wireless keyboard

https://github.com/maricn/interception-vimproved#running 
TADA68 is a external keyboard,how do i find the external keyboard name

Thanks

**Comment by maricn** [comment]
**tl;dr;**
I'm not at a linux machine that i have hardware access to atm, but `udevadm monitor --property` and plugging out and plugging back in your keyboard should give you something to work with.

**More Info**

InterceptionTools is the underlying foundation and the `interception-vimproved` is merely a "plugin". They have described what you're interested in at https://gitlab.com/interception/linux/tools#how-it-works. I deliberately wouldn't go into details in README.md here, but it could be cleaned up a bit.

* Basics of udev in linux: https://www.linuxfordevices.com/tutorials/linux/udevadm-command
* As usual, Arch wiki has in depth page on udev: https://wiki.archlinux.org/title/Udev
